lib/file-perms.h

```diff
@@ -51,5 +51,6 @@ gboolean file_perm_options_apply_file(const FilePermOptions *self, gchar *name);
 gboolean file_perm_options_apply_dir(const FilePermOptions *self, gchar *name);
 gboolean file_perm_options_apply_fd(const FilePermOptions *self, gint fd);
 gboolean file_perm_options_create_containing_directory(const FilePermOptions *self, gchar *name);
+gboolean file_perm_options_check_file_is_device(const gchar *name);
 
```

Comment:
hmm... are you sure that this is the right place for this function?
I mean `check_file_is_device` is independent from `FilePermOptions`.
(If FilePermOptions is a class, then this function is not a method of that class...)